backend/functions/routes/coords.js

```diff
@@ -0,0 +1,76 @@
+// Set up Firebase functions
+const functions = require('firebase-functions');
+
+// Set up Firebase Admin for Firestore
+const admin = require('firebase-admin');
+admin.initializeApp(functions.config().firebase);
+let db = admin.firestore();
```

Comment:
Isn't there a way to hide the access to the firestore behind some sort of function? Maybe having this `admin` name in our application is not a good sign as we do not want to give admin powers to anyone using our app.

Reply:
I don't think I understand what you mean here.